generic/interbench.py.data/makefile_fix.patch

```diff
@@ -0,0 +1,20 @@
+--- interbench-0.31/Makefile.orig	2016-06-29 08:33:47.607237136 -0400
++++ interbench-0.31/Makefile	2016-06-29 08:35:48.088034108 -0400
+@@ -1,10 +1,9 @@
+-CC=gcc
+-CFLAGS=-W -Wall -g -O2 -s -pipe
+-LDFLAGS=-lrt -lm -pthread
+-
+-interbench: interbench.o hackbench.o
+-interbench.o: interbench.c
+-hackbench.o: hackbench.c
+-
++C ?= gcc
++CFLAGS=-W -Wall -g -lpthread -O2 -s -pipe
++LDFLAGS=-lrt -lm -lpthread
++default: 
```

Comment:
Nice, you removed the empty chars :smile:. Only the `C ?= gcc` is still unhappy as the rest of the lines are laughing to it... :laughing: